tests/functional/adapter/incremental_strategies/test_incremental_strategies.py

```diff
@@ -0,0 +1,121 @@
+import pytest
+
+from dbt.tests.util import run_dbt, check_relations_equal
+from dbt.tests.adapter.simple_seed.test_seed import SeedConfigBase
+from tests.functional.adapter.incremental_strategies.fixtures import *
+
+class TestIncrementalStrategies(SeedConfigBase):
```

Comment:
This might be better classed as a `Base` class over `Test` as it setup and we don't necessarily want anything in it being kicked off by our process which looks for `test` in either class or method names to run

Reply:
curious why we are taking this in `SeedConfigBase`